Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "avr-progmem"
-version = "0.3.3"
```

Comment:
Upgrading `ufmt` is a breaking change, so we need to bump the version to 0.4.0. Thus, the PR should also target the new `v0.4` branch instead.

Reply:
Updated